stdlib/websub/src/main/ballerina/src/websub/Module.md

```diff
@@ -127,385 +104,26 @@ A hub client endpoint is also made available to publishers and subscribers to pe
 - Subscribers
   - Subscribe/Unsubscribe to/from topics at a hub
 
-## Samples
-This sample demonstrates a Subscriber Service with `subscribeOnStartUp` set to true. This will result in a
```

Comment:
Do **NOT** remove these theoretical concepts? Rename the section as I have proposed above add following sentence to L131

> When the `subscribeOnStartUp` is set to true in the Subscriber Service, it will result in a